data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
I fire the objectives action when it should change and use silent= to keep quiet about it. Or, I should .. it seemed to work when I tested but i may have misses a path through the code.